chat-plugins/info.js

```diff
@@ -636,6 +646,33 @@ var commands = exports.commands = {
 		"The parameter 'mega' can be added to search for Mega Evolutions only, and the parameters 'FE' or 'NFE' can be added to search fully or not-fully evolved Pokemon only.",
 		"The order of the parameters does not matter."],
 
+	randpoke: 'randompokemon',
+	randompokemon: function (target) {
+		var targets = target.split(",");
+		var targetsBuffer = [];
+		var qty;
+		for (var i = 0; i < targets.length; i++) {
+			if (!targets[i]) continue;
+			var num = Number(targets[i]);
+			if (Number.isInteger(num)) {
+				if (qty) return this.sendReply("Only specify the number of Pokemon once.");
+				qty = num;
+				if (1 > qty || qty > 15) return this.sendReply("Number of random Pokemon must be between 1 and 15.");
+				targetsBuffer.push("random" + qty);
+			} else {
+				targetsBuffer.push(targets[i]);
+			}
+		}
+		if (!qty) targetsBuffer.push("random1");
+		//Reassigns the target variable to be called in /dexsearch
+		var argumentsBuffer = arguments;
+		argumentsBuffer["0"] = targetsBuffer.join(",");
```

Comment:
I'd argue the only reason ``apply`` begun to be used is that it allowed a one-line implementation of the function for ``/details`` (bad reason imo, but that's another story), without using any named arguments.

However, as of 5343b9cc812f5a4168ea4e97e3141c90d8506480, it's no longer a one-liner, and it now uses named arguments. Therefore, I think we should go back to our prior condition of Law of Demeter's respect. Also, the commands API is cool and very newcomer-friendly. Let's not obfuscate it.